perses/tests/test_repex.py

```diff
@@ -80,7 +80,7 @@ def test_RESTCapableHybridTopologyFactory_repex_neutral_mutation():
 
         DDG = abs(data['ala-thr']['free_energy'] - data['thr-ala']['free_energy'] * -1)
         dDDG = np.sqrt(data['ala-thr']['error'] ** 2 + data['thr-ala']['error'] ** 2)
-        assert DDG < 6 * dDDG, f"DDG ({DDG}) is greater than 6 * dDDG ({6  * dDDG})"
+        assert DDG < 3 * dDDG, f"DDG ({DDG}) is greater than 3 * dDDG ({3  * dDDG})"
```

Comment:
@ijpulidos : Why did you change this from 6 to 3? I think we want to keep it as 6, based on these results: 
https://github.com/choderalab/perses/issues/1044#issuecomment-1157006442
https://github.com/choderalab/perses/issues/1044#issuecomment-1167913833

Reply:
I see, I remember discussing about making it 3*dDDG. So, would this be 6*dDDG for the neutral test and 3*dDDG for the charge mutation test or should we just keep 6 for both?

Reply:
I think we'd want to keep 6 for both. I also was thinking you could open a separate PR for speeding up the neutral mutation tests, although its probably fine to make them here -- you'd just need to update the description and title of this PR -- up to you.

Reply:
Yeah, I think it is better to make them here, since we are changing the same file, this could avoid potential merge conflicts. I'll update the description.